packages/ui/package.json

```diff
@@ -20,6 +20,7 @@
     "primevue": "^3.41.0",
     "vee-validate": "^4.11.8",
     "vue": "^3.3.4",
+    "vue-multiselect": "^3.0.0-beta.3",
```

Comment:
doesn't prime vue come with it's own multiselect as well?

Reply:
It does however in my opinion I don't think its as comprehensive as Vue Multiselect. Although I think it's something that we could explore removing in the future, I think given that every RailSmart/Forge project uses multiselects heavily, it's important that there is the correct investigation done to make sure that they do offer the same functionality before removing it.